hs_core/tasks.py

```diff
@@ -250,3 +250,15 @@ def create_bag_by_irods(resource_id):
     else:
         logger.error('Resource does not exist.')
         return False
+
+
+@shared_task
+def check_irods_files():
+    """
+    Check every resource for iRODS file problems.
+
+    This logs ERROR messages for every problem it finds
+    """
+    for r in BaseResource.objects.all():
+        r.check_irods_files(log_errors=True, echo_errors=False,
+                            stop_on_error=False, return_errors=False)

```

Comment:
@alvacouch This is not addressed either. I suggest just to remove this from celery task, but run via management command.